pre_commit/error_handler.py

```diff
@@ -26,6 +28,11 @@ def _to_bytes(exc):
 
 def _log_and_exit(msg, exc, formatted):
     error_msg = b''.join((
+        _to_bytes('### version information\n'),
+        _to_bytes('pre-commit.version={}\n'.format(C.VERSION)),
+        _to_bytes('sys.version={}\n'.format(sys.version.replace('\n', ' '))),
+        _to_bytes('sys.executable={}\n'.format(sys.executable)),
+        _to_bytes('### error information\n'),
```

Comment:
while we're at it (since we're markdown formatting this) let's put fixed-width code braces around the two chunks of output so it's easier to copy paste the error log into an issue (I hinted at this in the previous review but probably didn't explicitly ask for it there -- my b)